Annotations/Anonymize.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace SuperBrave\GdprBundle\Annotations;
+
+use Doctrine\Common\Annotations\Annotation;
+use Doctrine\Common\Annotations\AnnotationException;
+
+/**
+ * Annotation to flag an entity field to be anonymized
+ * to conform with the GDPR right to be forgotten
+ * 
+ * @Annotation()
+ * @Annotation\Target({"PROPERTY"})
+ */
+class Anonymize
+{
+    /**
+     * Alias of TYPE_CONCAT
+     */
+    const TYPE_STRING = 'string';
+
+    /**
+     * Anonymize a string to <columnName>-<primaryKey>
+     */
+    const TYPE_CONCAT = 'concat';
+
+    /**
+     * Anonymize a field by emptying it
+     * Alias of TYPE_FIXED with an empty value.
+     */
+    const TYPE_TRUNCATE = 'truncate';
+
+    /**
+     * Anonymize an IP field by filling in 127.0.0.1
```

Comment:
Is this something you can solve with the fixed value instead of describing it here in what the type will resolve?

Reply:
Is there a chance the IP field can be anything other then a string?

Because if not then there is indeed no need to have a seperate type for doing essentially the same thing.

Reply:
That depends on the end-user. With `TYPE_FIXED` the end-user can fill in whatever he wants. The negative part is that for every ip field you have to manually fill in the fixed value.

Should we add the possibility to allow the end-user to come up with his on types based on compiler pass services?

Reply:
This is something @joostvanveelen wanted to implement. So that is in the pipelines already.

Reply:
Google or GA in this part is masking the ip-adres by setting the last bytes of a ip-adres to 0: https://support.google.com/analytics/answer/2763052?hl=nl. I personally think this is a better solution than setting all ip-addresses to localhost. If we all think this is the case then this type should available.

Reply:
Yes, absolutely. Keep in mind while implementing that IP addresses can also be IPV6 addresses.